Meta/Keywords/scripting_library/TA/trigger/eval_triggered.py

```diff
@@ -0,0 +1,244 @@
+#  Drakkar-Software OctoBot-Trading
+#  Copyright (c) Drakkar-Software, All rights reserved.
+#
+#  This library is free software; you can redistribute it and/or
+#  modify it under the terms of the GNU Lesser General Public
+#  License as published by the Free Software Foundation; either
+#  version 3.0 of the License, or (at your option) any later version.
+#
+#  This library is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+#  Lesser General Public License for more details.
+#
+#  You should have received a copy of the GNU Lesser General Public
+#  License along with this library.
+
+import octobot_commons.constants as commons_constants
+import octobot_commons.errors as commons_errors
+import octobot_commons.enums as commons_enums
+import octobot_commons.dict_util as dict_util
+import octobot_evaluators.matrix as matrix
+import octobot_evaluators.enums as evaluators_enums
+import octobot_tentacles_manager.api as tentacles_manager_api
+import octobot_trading.modes.script_keywords.basic_keywords as basic_keywords
+import tentacles.Meta.Keywords.scripting_library.UI.inputs.triggers as triggers
+
+
+def _is_first_candle_only(context):
+    if not context.exchange_manager.is_backtesting:
+        # this is a backtesting only optimization
+        return False
+    tentacle_config = context.tentacle.trading_config if hasattr(context.tentacle, "trading_config") \
+        else context.tentacle.specific_config
+    return tentacle_config.get(triggers.TRIGGER_ONLY_ON_THE_FIRST_CANDLE_KEY, False)
+
+
+def _is_first_candle_call(context, init_key):
+    # TODO: figure out if we currently are in the 1st call of the given candle (careful with timeframes)
+    return not context.symbol_writer.are_data_initialized_by_key.get(init_key, False)
+
+
+async def evaluator_get_result(
+        context,
+        tentacle_class,
+        time_frame=None,
+        symbol=None,
+        trigger=False,
+        value_key=commons_enums.CacheDatabaseColumns.VALUE.value,
+        cache_key=None,
+        config_name=None,
+        config: dict = None
+):
```

Comment:
Should we add typing to keywords method so that user will have a better autocompletion?